package.json

```diff
@@ -30,7 +30,7 @@
   "scripts": {
     "autobuild": "rollup -c -w",
     "build": "rollup -c",
-    "dev": "cross-env NODE_ENV=test karma start ---auto-watch --no-single-run --browsers chrome --grep",
+    "dev": "karma start ---auto-watch --no-single-run --browsers chrome --grep",
```

Comment:
Removed so istanbul does not litter the code with coverage reporting.

Reply:
What was the issue? I'm not sure what you mean about littering the code with coverage reporting

Reply:
istanbul adds counters to every method, so the code is not very readable. its needed for coverage reporting (and is done in npm test), but its useless in dev mode.